alpha: 1.0
beta: 2.0
gamma: 0.6
delta: 1.0
max_linear_acceleration: 2.0
max_angular_acceleration: 5.0
min_linear_velocity: 0.0
max_linear_velocity: 1.0
min_angular_velocity: -1.5
max_angular_velocity: 1.5
time_trajectory: 1.0
trajectory_sample: 10
velocity_sample: 7
sample_time: 0.1
map_name: empty
laser_topic: base_scan
position_topic: base_pose_ground_truth
velocity_topic: odom
cmd_vel_topic: cmd_vel
robot_width: 0.6
robot_length: 0.9
goal_tolerance: 0.1